workflows/nextflow.config

```diff
@@ -22,6 +22,15 @@ profiles{
       }
       withLabel: bigdisk {
         queue = 'nextflow-batch-bigdisk-queue'
+        cpus = 8
+        memory = { 120.GB * task.attempt}
+        errorStrategy = {task.exitStatus in 137..140 ? 'retry' : 'terminate' }
+        maxRetries = 1
```

Comment:
This should probably be put in a separate label (though as I said above, if it is only one process, we might not want to use a label/config). I wanted to keep the queue specification separate from cpu & memory, as there may be processes that need a large amount of disk space but don't need the memory.